### PR TITLE
feat: adding pending property to message

### PR DESCRIPTION
**Pull request by vishalnarkhede**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -180,8 +196,9 @@ func (c *Client) GetAppSettings(ctx context.Context) (*AppResponse, error) {
 
 // UpdateAppSettings makes request to update app settings
 // Example of usage:
-//  settings := NewAppSettings().SetDisableAuth(true)
-//  err := client.UpdateAppSettings(settings)
+//
+//	settings := NewAppSettings().SetDisableAuth(true)
+//	err := client.UpdateAppSettings(settings)
```
Comment (yaziine) [inline app.go:201]: what's the need of this change?

File: channel_type_test.go [hunk 2]
```diff
@@ -55,6 +55,22 @@ func TestClient_ListChannelTypes(t *testing.T) {
 	assert.Contains(t, resp.ChannelTypes, ct.Name)
 }
 
+func TestClient_UpdateChannelTypeMarkMessagesPending(t *testing.T) {
+	c := initClient(t)
+	ct := prepareChannelType(t, c)
+	ctx := context.Background()
+
+	// default is on
```
Comment (yaziine) [inline channel_type_test.go:63]: off?

**Review by yaziine — DISMISSED** [review]
LGTM (only two minor comments)

File: app.go (hunk 1 above)
Comment (vishalnarkhede) [reply app.go:201]: I think linter did that

**Pull request merged**